DVTShowcase/AppDetailViewController.swift

```diff
@@ -0,0 +1,36 @@
+
+
+import UIKit
+
+class AppDetailViewController: UIViewController {
+
+    private var selectedApp: [String: AnyObject?]
+    
+    //MARK: IBOutlets
+    
+    @IBOutlet weak var nameContentView: UIView!
+    @IBOutlet weak var galleryContentView: UIView!
+    @IBOutlet weak var descriptionContentView: UIView!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationItem.title = selectedApp["name"] as? String
+
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    init(with dvtApp:[String: AnyObject?]) {
```

Comment:
Why is this a dictionary? can you not just pass the fail why do you need to pass the whole dictionary to the next viewcontroller